### ISSUE TITLE
Add a py.typed file for submodule

### ISSUE THREAD

**Issue by crwilcox** [googleapis/python-bigquery#1019]
To be in accordance with [PEP 561](https://www.python.org/dev/peps/pep-0561/) we should add a py.typed file to google/cloud/bigquery/py.typed

The file can be empty but does need to be present for some type checkers.

**Comment by tswast** [comment]
Please don't until https://github.com/googleapis/python-bigquery/issues/987 is fixed.

**Comment by plamut** [comment]
Mypy is happy now, unblocking.

We will only add `py.typed` to the `v3` branch, though, as discussed offline.

**Comment by plamut** [comment]
Closed in #1058 (merged into the `v3` branch).
